airflow/www/templates/airflow/master.html

```diff
@@ -19,6 +19,14 @@
 
 {% extends 'appbuilder/baselayout.html' %}
 
+{% block page_title -%}
+  {% if title is defined -%}
+    {{ title }} - {{ app_name }}
```

Comment:
Isn't this going to show "DAGs - Airflow - Airflow"? for the task.html template?

It sounds like it is probably worth defining a `title` block in every page, that way we don't need the app_name in all of them.

Reply:
Would this be defined in `master.html` @ashb? I think this might be the more robust option, as right now I have modified several files to update the site_title variable. It works nicely as is, but the above is cleaner I feel.

Reply:
@mik-laj any thoughts?